src/commands/Fun/initial.js

```diff
@@ -0,0 +1,12 @@
+exports.run = function(bot, msg, args) {
+    if (args.length < 1) {
+        throw new Error('You must input text to be transformed.');
```

Comment:
Eh... throwing strings is considered bad practice because they dont generate stack traces. I guess if you really want I can do that though.